### PR TITLE
Add/schedule

### PR DESCRIPTION
**Pull request by martinheidegger**
This PR adds the current schedule to the page.

I checked quite a few things but some things are missing

- [ ] Browser-testing: Works on IE/Windows/Mobile
- [ ] Lighting talks: Currently the Lightning talks don't show 
      anything. The speakers with lightning talk entries also have no entry. 
- [ ] Resizing of window: Does it work well? x-platform?
- [ ] Entry-click-area: The buttons in the schedule are a bit weird
- [ ] The descriptions of the entries have not been properly formatted yet.
      I think a markdown parser is necessary.

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
huuuuuuuuge

**Comment by yosuke-furukawa** [comment]
1. I don't want browser test, need quick update.
2. instead of, please add demo gif animation on this issue.
3. I like small chages, no need markdown parser, easy click area.

anyway, please separate timetable PR and speaker page PR.

**Comment by yosuke-furukawa** [comment]
Thank you @martinheidegger , @leader22 .
I would like to update speaker / timetable ASAP.

I keep this PR open to review well. 
However We need speaker information for attendees. 

@leader22 please update website and attach timetable spreadsheet link if you have time.

**Comment by martinheidegger** [comment]
> anyway, please separate timetable PR and speaker page PR.

I think the speaker page is already published?! This just modifies the speakers page to use the same module.

File: 2018/src/confcal/interactive.confcal [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[Room A]
+10:00-10:30 Doors Open
+10:30-11:45 Node/JS Discussion
+13:00-15:00 Code And Learn by hiroppy
+15:00-17:00 JavaScript TodoMVC Workshop by Yosuke Furukawa
+17:00-18:00 Lightning Talks
+    - Serverless in Scala on Node.js: Get Fast and Productive by Tatsuno Yasuhiro
+    - 私の生活を変えたHeadless Chrome by 大島　光貴
+    - Deploy TypeScript with CodePipeline in Lambda by 石川　将吾
```
Comment (kakts) [inline 2018/src/confcal/interactive.confcal]: The Speaker wants to change the title as below.
Would you change it?


```
before: Deploy TypeScript with CodePipeline in Lambda
after:    Deploy TypeScript with CodePipeline in Fargate
```

Comment (martinheidegger) [reply 2018/src/confcal/interactive.confcal]: Done in https://github.com/nodefest/nodefest.github.com/pull/139#pullrequestreview-166411996

**Comment by yosuke-furukawa** [comment]
Thank you Very much. it is very helpful, But this pr is so huge, i don't have time to judge this PR is maintainable.

**Comment by martinheidegger** [comment]
I can make some effort and break it into a fewer parts.

**Comment by Farid212** [comment]
ie vs chrome


Internet Explorer does not react like in chrome the featured speaker is a little bit weird
hope you get that picture

**Comment by martinheidegger** [comment]
@Farid212 Thank you for testing. Indeed: IE doesn't support coloring. This can be fixed using pre-processing of the images. This is not related to this PR.

File: 2018/gulpfile.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 'use strict';
-const fs = require('fs');
+const fse = require('fse');
```
Comment (yosuke-furukawa) [inline 2018/gulpfile.js]: what is fse ? I would like to be simple, fs is better, if you need promise, use fs.promises or bluebird instead.

File: 2018/gulpfile.js [hunk 3]
```diff
@@ -12,12 +12,14 @@ const source = require('vinyl-source-stream');
 const buffer = require('vinyl-buffer');
 const uglify = require('gulp-uglify');
 const del = require('del');
+const confCal = require('conf-cal');
```
Comment (yosuke-furukawa) [inline 2018/gulpfile.js]: conf-cal is a csv parser?

File: 2018/index.html [hunk 4]
```diff
@@ -114,7 +114,7 @@ <h2 class="home-subhead">Featured Speakers</h2>
         
         <li>
           
-<a class="speaker" data-key="addaleax">
+<a class="speaker" href="speakers.html#speaker-addaleax">
```
Comment (yosuke-furukawa) [inline 2018/index.html]: Could we delete this `data-key` ??

File: 2018/gulpfile.js (hunk 3 above)
Comment (martinheidegger) [reply 2018/gulpfile.js]: Conf-cal is a parser for the conf-cal textformat parser.

I made the conf-cal concept to make it easy for us to maintain: Here is the raw-data used:

[`conference.confcal`](https://github.com/nodefest/nodefest.github.com/blob/e24664e1e2f08d9cf9bd2d862ff43ebdca61b545/2018/src/confcal/conference.confcal#L26)
[`interactive.confcal`](https://github.com/nodefest/nodefest.github.com/blob/e24664e1e2f08d9cf9bd2d862ff43ebdca61b545/2018/src/confcal/interactive.confcal#L26)

File: 2018/index.html (hunk 4 above)
Comment (martinheidegger) [reply 2018/index.html]: Yes, i updated the overlay system to accomodate for it.

**Comment by yosuke-furukawa** [comment]
Could you attach current screenshot?

File: 2018/gulpfile.js (hunk 2 above)
Comment (martinheidegger) [reply 2018/gulpfile.js]: [`fs.promises`](https://www.npmjs.com/package/fs.promises) seems less well maintained than [`fse`](https://www.npmjs.com/package/fse) ?!

Comment (yosuke-furukawa) [reply 2018/gulpfile.js]: nope, fs.promises is a new feature for node.js

Comment (yosuke-furukawa) [reply 2018/gulpfile.js]: https://nodejs.org/api/fs.html#fs_fs_promises_api

Comment (martinheidegger) [reply 2018/gulpfile.js]: Gulp 3.9.1 is broken for Node.js 10 https://github.com/nodejs/node/issues/19786#issuecomment-379583525

I would need to update Gulp to the 4 beta to use `fs.promises`

**Comment by martinheidegger** [comment]
Conference Calendar

Interactive Calendar

Speaker Overlay

Screen-capture of the schedule

**Pull request merged**